managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -3258,6 +3375,8 @@ void notifyEntriesAvailable() {
             }
 
             PENDING_READ_OPS_UPDATER.incrementAndGet(this);
+            // 触发监听的时候，就只修改了原来记录的op里面的readPosition
+            // op的maxReadPosition没有变，那不就寄了？？
```

Comment:
please translate the comment to English

Reply:
has fix. forget to delete the comment. sry